.github/workflows/sonarqube.yml

```
@@ -0,0 +1,67 @@
+name: CI Pipeline
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)